posthog/queries/paths.py

```diff
@@ -65,11 +65,7 @@ def calculate_paths(self, filter: PathFilter, team: Team):
         sessions = (
             Event.objects.add_person_id(team.pk)
             .filter(team=team, **(event_filter), **date_query)
-            .filter(
-                ~Q(event__in=["$autocapture", "$pageview", "$identify", "$pageleave", "$screen"])
-                if event is None
-                else Q()
-            )
+            .filter(~Q(event__in=["$pageview", "$identify", "$pageleave", "$screen"]) if event is None else Q())
```

Comment:
Wouldn't we want to keep `"$autocapture"` in here, since it's a Negation query? i.e. Event not in `[this list]`.

Reply:
yeah, misread